Algorithms/Algorithms/HuffmanCoding/HuffmanCoding.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms.HuffmanCoding
+{
+
+    public class HuffmanNode : IComparable<HuffmanNode>
+    {
+        public long frequency;
```

Comment:
Please follow the [naming guideline](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines)

Reply:
Also please rename "ch" to something more readable, like "character" or "symbol".